base/src/org/adempiere/engine/CostEngine.java

```diff
@@ -509,11 +509,9 @@ private BigDecimal convertCostToSchemaCurrency(MAcctSchema acctSchema , IDocumen
 				model.getC_Currency_ID(), acctSchema.getC_Currency_ID() ,
 				model.getDateAcct(), model.getC_ConversionType_ID() ,
 				model.getAD_Client_ID(), model.getAD_Org_ID());
-		if (rate != null) {
+		if (rate != null) 
```

Comment:
This implementation change the rounding standard based on the currency costing rounding  that use all System